contrib_bots/contrib_server.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
+from flask import Flask, request, jsonify
+from run import get_lib_module
+from bot_lib import BotHandlerApi, StateHandler
+import os
+import sys
+import json
+from typing import Any, Dict, Mapping, Union
+from werkzeug.exceptions import BadRequest
+from six.moves.configparser import SafeConfigParser
+
+our_dir = os.path.dirname(os.path.abspath(__file__))
+
+# For dev setups, we can find the API in the repo itself.
+if os.path.exists(os.path.join(our_dir, '../api/zulip')):
+    sys.path.insert(0, '../api')
+
+from zulip import Client
+
+config_file_path = sys.argv[1]
+bots_config = {} # type: Dict[str, Mapping[str, str]]
+
+parser = SafeConfigParser()
+parser.read(config_file_path)
+
+for section in parser.sections():
+    bots_config[section] = {
+        "email": parser.get(section, 'email'),
+        "key": parser.get(section, 'key'),
+        "site": parser.get(section, 'site'),
+    }
+
+available_bots = bots_config.keys()
+bots_lib_module = {} # type: Dict[str, Any]
+
+for bot in available_bots:
+    path = "bots/" + str(bot) + "/" + str(bot) + ".py"
+    bots_lib_module[bot] = get_lib_module(path)
+
+app = Flask(__name__)
+
+@app.route('/')
+def main():
```

Comment:
Just checking, is it necessary to define this route / function?

If so, can it be called something besides `main()`?
If not, can it be removed?